dvclive/data/image_pil.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+from PIL.Image import Image
+
+from .base import Data
+
+
+class ImagePIL(Data):
+    subdir = "images"
+    suffixes = [".jpg", ".jpeg", ".gif", ".png"]
+
+    @staticmethod
+    def could_log(val: object) -> bool:
+        if isinstance(val, Image):
+            return True
+        return False
+
+    @property
+    def output_path(self) -> Path:
+        if Path(self.name).suffix not in self.suffixes:
+            raise ValueError(
+                f"Invalid image suffix '{Path(self.name).suffix}'"
+                f" Must be one of {self.suffixes}"
+            )
+        return self.output_folder / self.subdir / self.name
+
+    def dump(self, val, step) -> None:
+        super().dump(val, step)
+        output_path = Path(str(self.output_path).format(step=step))
+        output_path.parent.mkdir(exist_ok=True, parents=True)
+
+        val.save(output_path)
+
+    @property
+    def summary(self):
+        return {}
```

Comment:
Should image be reflected in the summary?

Reply:
We could provide the URL of the latest.